### PR TITLE
Updating the layout

### PR DESCRIPTION
**Pull request by jeroenpraat**
Found a way to nicely show Play Store and F-Droid links

### CODE REVIEW

File: Using-Mastodon/Apps.md [hunk 1]
```diff
@@ -2,131 +2,130 @@ List of apps
...
-|[mastty](https://play.google.com/store/apps/details?id=com.sasaneko.mastodon)|N/A|White Cat Factory|
-|[Pawoo](https://play.google.com/store/apps/details?id=jp.pxv.pawoo)|N/A|[@pixiv@pawoo.net](https://pawoo.net/users/pixiv)|
-|[Stegodon](https://play.google.com/store/apps/details?id=com.gilawhost.stegolophodon)|N/A|[@GilaW@mastodon.cloud](https://mastodon.cloud/@Gilaw)|
-|[Tootaloo](https://play.google.com/store/apps/details?id=net.liverworks.tootalooa)|N/A|[liverworks.net](http://liverworks.net/)|
-|[Tootdon](https://play.google.com/store/apps/details?id=club.tootdon.app)|N/A/|[@tootdon@mstdn.jp](https://mstdn.jp/@tootdon)|
-|[Troutoss](https://play.google.com/store/apps/details?id=jp.gr.java_conf.miwax.troutoss)|N/A|[@tomoya0x00@mstdn.jp](https://mstdn.jp/@tomoya0x00)|
+|**11t**<br />([Play](https://play.google.com/store/apps/details?id=com.jeroensmeets.mastodon))|<https://github.com/jeroensmeets/mastodon-app>|[@jeroensmeets@mastodon.social](https://mastodon.social/users/jeroensmeets)|
+|**[AndStatus](http://andstatus.org/)**<br />([Play]()/[F‑Droid]())|<https://github.com/andstatus/andstatus>|[@AndStatus@mastodon.social](https://mastodon.social/@AndStatus)|
```
Comment (nightpool) [inline Using-Mastodon/Apps.md]: If this one doesn't have Play/F-Droid links maybe you should leave them out?

Comment (nightpool) [reply Using-Mastodon/Apps.md]: Also all the others have the links removed but this one is still bold and linked.

File: Using-Mastodon/Apps.md [hunk 2]
```diff
@@ -2,131 +2,130 @@ List of apps
...
-|[friends.nico](https://itunes.apple.com/us/app/friends-nico/id1230158182)|N/A|[@friends_nico@friends.nico](https://friends.nico/@friends_nico)|
-|[Tootter3.0](https://itunes.apple.com/us/app/id1229555793)|N/A|[@Tootter_for_Mastodon@mstdn.jp](https://mstdn.jp/@Tootter_for_Mastodon)|
+|**11t**<br />([iTunes](https://itunes.apple.com/nl/app/11t-for-mastodon/id1180628189))|<https://github.com/jeroensmeets/mastodon-app>|[@jeroensmeets@mastodon.social](https://mastodon.social/users/jeroensmeets)|
+|**Amaroq**<br />([iTunes](https://itunes.apple.com/us/app/amarok-for-mastodon/id1214116200?ls=1&mt=8))|<https://github.com/ReticentJohn/Amaroq>|[@eurasierboy@mastodon.social](https://mastodon.social/users/eurasierboy)|
+|**iMast**<br />([iTunes](https://itunes.apple.com/jp/app/imast/id1229461703))|<https://github.com/cinderella-project/iMast>|[@imast_ios@imastodon.net](https://imastodon.net/@imast_ios)|
+|**TootyFruity**<br />([iTunes](https://docs.google.com/forms/d/e/1FAIpQLScW2lvZaKrOm4AKJH5HLI_Ul0Yr3CsXulf5bcTJQ1xaN5jiPg/viewform?c=0&w=1)) *(beta)*|<https://github.com/eggplantPrince/tootyFruity>|[@eggplant@mastodon.social](https://mastodon.social/users/eggplant)|
```
Comment (nightpool) [inline Using-Mastodon/Apps.md]: this one is marked beta but also bolded.

File: Using-Mastodon/Apps.md [hunk 3]
```diff
@@ -2,131 +2,130 @@ List of apps
...
+|**Tooter**|[Chrome(ium)](https://chrome.google.com/webstore/detail/tooter/okmlpjijminjkikninbkcnfmhkofgnnk)/[Firefox](https://addons.mozilla.org/nl/firefox/addon/tooter/)|<https://github.com/ineffyble/tooter>|[@effy@mastodon.social](https://mastodon.social/users/effy)|
 
 ## Bridges from/to other platforms
 
 |App|Platform|Source code|Developer(s)|
 |---|--------|-----------|------------|
-|MastodonToTwitter|CLI|<https://github.com/halcy/MastodonToTwitter>|[@halcy@mastodon.social](https://mastodon.social/@halcy)|
+|**[Autopost to Mastodon](https://wordpress.org/plugins/autopost-to-mastodon/)**|Wordpress plugin|<https://github.com/L1am0/mastodon_wordpress_autopost>|[L1am0](http://l1am0.eu/)|
+|**MastodonToTwitter**|CLI|<https://github.com/halcy/MastodonToTwitter>|[@halcy@mastodon.social](https://mastodon.social/@halcy)|
 |Retoot|CLI|<https://gitlab.com/mojo42/retoot>|[@Mojo@apoil.org](https://apoil.org/users/mojo)|
```
Comment (nightpool) [inline Using-Mastodon/Apps.md]: this one wasn't bolded?

**Comment by jeroenpraat** [comment]
@nightpool Thanks for looking at it. I forgot those things you were talking about.

About linking a bold name: I only link those if they have their own website.

**Comment by jeroenpraat** [comment]
@nightpool Of course I can link those wothout a website to their README-files on Github. That's maybe a good idea to make things look more srtreamlined. 

Please give me time to fix this. I will mention you again when done.

**Comment by nightpool** [comment]
Oh, yeah, no worries! Just trying to close out some PRs here.

**Comment by jeroenpraat** [comment]
@nightpool Done!

**Comment by nightpool** [comment]
Looks great! this was a huge effort and it's improved things a lot. I'm gonna merge this, but there's also a quick tweak I want to make—when the *only* store linked is the play store, can you have it say "Play Store" instead of "Play"? I understand shortening it for multiple stores, but when it's the only one I think people might get confused about what the link is for and not understand what it is.

**Pull request merged**